### PR TITLE
Only publish release variants

### LINKED ISSUE #1150: User.logout() crashes app when minifyEnabled=true and debuggable=false (Proguard enabled)
#### SDK and version
SDK : realm-kotlin
Version: 'io.realm.kotlin:library-sync:1.5.0', 'io.realm.kotlin:gradle-plugin:1.5.0'

#### Observations
* How frequent do the crash occur? 
  - Reproducible
* Does it happen in production or during dev/test? 
  - Development
* Can the crash be reproduced by you?
  - Yes
* Can you provide instructions for how we can reproduce it?
  - Run app in debug or release with buildTypes config seen below in app build.gradle
  - Call User.logout() from a logged in user
  
  buildTypes {
        release {
            minifyEnabled true
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
            signingConfig signingConfigs.release
        }
        debug {
            minifyEnabled true
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
            debuggable false
            signingConfig signingConfigs.debug
        }
    }

#### Note
I need to manually place the lines from realm-kotlin's 'proguard-rules-consumer-common.pro'-file to my proguard-rules.pro-file. Otherwise, debug will not run with Proguard enabled.
Same issue: #942 

#### Crash log / stacktrace
#01 pc 0000000000d81a44  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #02 pc 0000000000d81c8c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #03 pc 0000000000d81db4  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #04 pc 00000000005a6ba0  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (void realm::util::terminate_with_info<char const*&>(char const*, int, char const*, char const*, char const*&)+116) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #05 pc 00000000005a6a54  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (realm::jni_util::JavaClass::get_jclass(_JNIEnv*, char const*)+104) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #06 pc 00000000005a6994  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (realm::jni_util::JavaClass::JavaClass(_JNIEnv*, char const*, bool)+52) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #07 pc 000000000058c3ec  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (app_complete_void_callback(void*, realm_app_error const*)+476) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.644 21115-21115 DEBUG                   pid-21115                            A        #08 pc 000000000060facc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #09 pc 000000000060f9f0  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #10 pc 000000000060f97c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #11 pc 00000000007b55bc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #12 pc 00000000007d5cec  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #13 pc 00000000007d5c34  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #14 pc 00000000007d5bf8  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #15 pc 000000000061ebfc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #16 pc 00000000007b7fd0  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #17 pc 00000000007d7c80  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #18 pc 00000000007d7c1c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #19 pc 00000000007d7bcc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #20 pc 00000000007bd498  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #21 pc 00000000007bd3ec  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #22 pc 00000000007bd3a8  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #23 pc 00000000007bd36c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #24 pc 000000000061ebfc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #25 pc 000000000061e7a4  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #26 pc 000000000061e648  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (realm_http_transport_complete_request+24) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #27 pc 000000000058dfac  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (complete_http_request(void*, _jobject*)+1604) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #28 pc 00000000005876d4  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/lib/arm64/librealmc.so (Java_io_realm_kotlin_internal_interop_realmcJNI_complete_1http_1request+56) (BuildId: e7b7749cd581d1516cf68c8de8acb9beaf7e2b07)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #31 pc 00000000002fd5c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (io.realm.kotlin.internal.interop.sync.ResponseCallbackImpl.response+18)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #33 pc 00000000002addd8  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fd.q$a.f+904)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #35 pc 000000000048d8fc  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+16)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #37 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #39 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #41 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #43 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #45 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #47 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #49 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #51 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #53 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #55 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #57 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #59 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #61 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #63 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #65 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #67 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.645 21115-21115 DEBUG                   pid-21115                            A        #69 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #71 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #73 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #75 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #77 pc 00000000002aca04  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.g+52)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #79 pc 00000000002ac7c2  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i.f+30)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #81 pc 00000000002ac78c  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (fc.i$a.r+48)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #83 pc 000000000048d92a  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (td.a.r+62)
2022-11-27 16:59:19.646 21115-21115 DEBUG                   pid-21115                            A        #85 pc 0000000000417c78  /data/app/~~EiwhyHPuPQzCFIZoOPNkqA==/com.crystalyze.crystalyze-cahkQ4HjzrIHvOVsnac7Uw==/oat/arm64/base.vdex (pg.n0.run+212)

### PR DESCRIPTION
**Pull request by rorbech**
Closes #1150

### CODE REVIEW

**Review by cmelchior — APPROVED** [review]
Looks good to me. Should we also add support for running minified unit tests in this PR. I suspect we could do it easily through a gradle property we can inject on Jenkins but otherwise just default to the current behavior?

**Comment by rorbech** [comment]
> Looks good to me. Should we also add support for running minified unit tests in this PR. I suspect we could do it easily through a gradle property we can inject on Jenkins but otherwise just default to the current behavior?

Yes that sounds like a good idea. I will give it a try.

File: packages/library-base/proguard-rules-consumer-common.pro [hunk 1]
```diff
@@ -102,6 +102,10 @@
     *;
 }
 
+-keep class kotlin.Unit {
```
Comment (nhachicha) [inline packages/library-base/proguard-rules-consumer-common.pro:105]: Why is this necessary?

File: packages/library-base/build.gradle.kts [hunk 2]
```diff
@@ -171,6 +171,9 @@ android {
     }
 
     buildTypes {
+        getByName("debug") {
+            consumerProguardFiles("proguard-rules-consumer-common.pro")
```
Comment (nhachicha) [inline packages/library-base/build.gradle.kts:175]: We apply the same rule for `debug` and `release`?

File: packages/library-base/proguard-rules-consumer-common.pro (hunk 1 above)
Comment (rorbech) [reply packages/library-base/proguard-rules-consumer-common.pro:105]: This is the return type for `Function0`. See https://github.com/realm/realm-kotlin/issues/1150#issuecomment-1348535477

File: packages/library-base/build.gradle.kts (hunk 2 above)
Comment (rorbech) [reply packages/library-base/build.gradle.kts:175]: Because the proguard consumer file also needs to be packed into the `debug` artifact if we for some reason should choose to minify it.
